CHANGES_NEXT_RELEASE

```diff
@@ -3,3 +3,5 @@ Allow get list of commands without sending measures (empty payload) [#256]
 Fix transport in autoprovision device depending on binding [#257]
 Fix: defaultKey in config.js (supposely fixing #222 and #207)
 Fix default resource /iot/json instead of /iot/d in config and tests
+Add: allow NGSIv2 for updating active attributes at CB, through configuration (#250)
+Add: NGSIv2 support for data transformation plugins (#250)
```

Comment:
What "data tranformation plugins" mean in this context? The ones at https://github.com/telefonicaid/iotagent-node-lib#-data-mapping-plugins? Wasn't that adaptation done in the context of the iotagent-node-lib?

Just to understand the content of the CNR... :)

Reply:
Yes, These are the data transformation plugins. As you can see, there are no real modifications in this PR with respect to the plugins. I just introduced this statement to reflect that NGSIv2 is supported.

Reply:
I understand... Whata about merging the two lines into one saying the following?

```
Add: allow NGSIv2 for updating active attributes at CB, through configuration based on iotagent-node-lib (#250)
```

That way it makes clear that it isn't any real modifications in the IOTAgent code itself with respect to the plugins.

Reply:
Fixed in e95216c36df695c6d4a2a097d4bc8cace20b8625